extensions/test-src/org/pentaho/platform/plugin/services/importer/MetadataImportHandlerIT.java

```diff
@@ -1,18 +1,19 @@
-/*!
+/*
  * This program is free software; you can redistribute it and/or modify it under the
- * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * terms of the GNU General Public License, version 2 as published by the Free Software
  * Foundation.
  *
- * You should have received a copy of the GNU Lesser General Public License along with this
- * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * You should have received a copy of the GNU General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/gpl-2.0.html
```

Comment:
@AliaksandrDrebenchuk  please use the proper license header for extensions project